src/App.js

```diff
@@ -26,7 +31,38 @@ class App extends React.Component {
 	 
 	changedHandler(e) {
         this.setState({username: e.target.value});
```

Comment:
try to use the useState hook, there are much cleaner ways to handle input change